Dockerfile

```
@@ -1,21 +1,33 @@
 COPY package*.json ./
 RUN npm ci --omit=dev && npm cache clean --force
 
-# Copy the rest of the application
-COPY . .
 
-# Set permissions and switch to non-root user
+# Stage 2: Runtime image
+FROM mcr.microsoft.com/playwright:v1.60.0-jammy
+
+# Install dumb-init (signal handling) and curl (healthcheck)
+RUN apt-get update && apt-get install -y dumb-init curl \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)